### PR TITLE
Add a write command to bgutil

### PR DESCRIPTION
**Pull request by dpanth3r**
Initial implementation of #89

---
This change is [Reviewable](https://reviewable.io/reviews/criteo/biggraphite/151)

### CODE REVIEW

**Comment by iksaif** [comment]
LGTM. Merge when the tests are happy :)

File: biggraphite/cli/command_write.py [hunk 1]
```diff
@@ -0,0 +1,93 @@
+#!/usr/bin/env python
```
Comment (rudy-6-4) [inline biggraphite/cli/command_write.py]: looks, good can you add a test ?

Comment (dpanth3r) [inline biggraphite/cli/command_write.py]: true, I should add one. will do :-)

**Comment by iksaif** [comment]
Tests seem to be broken

**Comment by iksaif** [comment]
Only some minor still issues

```
biggraphite/cli/command_write.py:25:1: E302 expected 2 blank lines, found 1 [pep8]
biggraphite/cli/command_write.py:31:1: D102: Missing docstring in public method [pep257]
biggraphite/cli/command_write.py:75:1: D102: Missing docstring in public method [pep257]
biggraphite/cli/command_write.py:84:27: E251 unexpected spaces around keyword / parameter equals [pep8]
biggraphite/cli/command_write.py:85:26: E251 unexpected spaces around keyword / parameter equals [pep8]
biggraphite/cli/command_write.py:86:36: E251 unexpected spaces around keyword / parameter equals [pep8]
```

**Comment by iksaif** [comment]
I fixed the lint errors :)

**Pull request merged**